documentation/domains/Domain.md

```diff
@@ -233,12 +222,16 @@ The current status of the operation of the WebLogic domain. Updated automaticall
 
 | Name | Type | Description |
 | --- | --- | --- |
+| `auxiliaryImages` | Array of [Auxiliary Image](#auxiliary-image) | Optionally use auxiliary images to provide Model in Image model, application archive and WebLogic Deploy Tooling files. This is a useful alternative for providing these files without requiring modifications to the pod's base image 'domain.spec.image'. This feature internally uses a Kubernetes emptyDir volume and Kubernetes init containers to share the files from the additional images with the pod. |
```

Comment:
Optionally use -> Optionally, (comma) use
application archive -> application archive, (comma)
'domain.spec.image' -> `domain.spec.image` (code font; back ticks instead of straight quotes)